doc-content/jbpm-docs/src/main/asciidoc/BPMN2/proc-set-user-task-assignment-strategy.adoc

```diff
@@ -0,0 +1,30 @@
+[id='proc-set-user-task-assignment-strategy_{context}']
+
+= Setting the user task assignment strategy
+
+The user task assignment strategy is used to automatically assign the tasks to a suitable user. The assignment strategy allows more efficient task allocation based on the associated properties, such as potential owners, task priority, and task data. The default user task assignment strategy in Red Hat Process Automation Manager is `org.jbpm.task.assignment.strategy`. You can also explicitly define an assignment strategy for a user task in {CENTRAL}.
```

Comment:
This is just a system property, not a strategy. According to this [code](https://github.com/kiegroup/jbpm/blob/62f92c98f5d3f68f08fd519638259be97aa2d574/jbpm-human-task/jbpm-human-task-core/src/main/java/org/jbpm/services/task/assignment/impl/AssignmentServiceImpl.java#L58), the default strategy is org.jbpm.services.task.assignment.impl.strategy.PotentialOwnerBusynessAssignmentStrategy. Can you please double-check?

Reply:
Thanks @MarianMacik, I've changed the sentence to "`org.jbpm.task.assignment.strategy` is the system property for the user task assignment strategy in {PRODUCT}"